src/panels/usesSummaryPanel/UseSummaryPanel.test.tsx

```diff
@@ -0,0 +1,43 @@
+import { render, screen } from "@testing-library/react";
+import { Activities, Purposes } from "entities/IUse";
+import React from "react";
+import { UseSummaryPanel } from "./UseSummaryPanel";
+
+describe("UseSummaryPanel", () => {
+  let use: any;
+
+  beforeEach(() => {
+    use = {
+      activity: Activities.SailingVessel,
+      purpose: Purposes.Pleasure,
+    };
+  });
```

Comment:
This `use` as it appears in the `beforeEach` hook is only used in the first test, with the other two updating it for different uses.  Could this be in-lined to each test to improve readability?

Reply:
Have updated to remove the `beforeEach`